documentation/testnet/getting_started/02_deploy_and_execute.md

```diff
@@ -204,9 +204,10 @@ Transfer credits with a `credits.aleo` program execution.
 
 ```
 snarkos developer transfer <AMOUNT_TO_TRANSFER> --input-record <INPUT_RECORD> --recipient <RECIPIENT_ADDRESS>  --private-key <PRIVATE_KEY> --query "<http://localhost:3030>" --broadcast "<http://localhost:3030/testnet3/transaction/broadcast>"
-
 ```
 
+Note: Replace `transfer` with `transfer-private`, `transfer-public`, `-- transfer-private-to-public`,
+or `-- transfer_public_to_private` based on your use case.
```

Comment:
```suggestion
Note: Replace `transfer` with `transfer-private`, `transfer-public`, `-- transfer-private-to-public`,
or `-- transfer_public_to_private` based on your use case.
```
Based on our previous discussion I believe `-- transfer-public-to-private` is correct?

Reply:
Neither of those seem to work.. 
![Screen Shot 2023-06-28 at 1 20 48 PM](https://github.com/AleoHQ/welcome/assets/37510951/048eeab3-05c4-4a09-a927-314181b96053)

I'm on the latest testnet3 branch and am getting these errors when trying to run any of the transfer commands other than `transfer-private`.

Just wanted to make a note of that before committing your suggestion, but assuming that is correct due to consistency.

Reply:
Thanks for checking, you are correct those flags do not exist.